examples/backups.py

```diff
@@ -25,40 +26,98 @@
     }
 }
 
+pp = pprint.PrettyPrinter(indent=4)
+
 ovc = OVC(config)
 backups = ovc.backups
+machines = ovc.virtual_machines
+datastores = ovc.datastores
 
-print("\n\n get_all with default params")
+print("\nget_all with default params")
 all_backups = backups.get_all()
 count = len(all_backups)
 for backup in all_backups:
-    print(backup)
+    print(f"{backup}")
+    print(f"{pp.pformat(backup.data)} \n")
 
 print("\nTotal number of backups {}".format(count))
 backup_object = all_backups[0]
 
-print("\n\n get_all with filers")
+print("\nget_all with filers")
 all_backups = backups.get_all(filters={'name': backup_object.data["name"]})
 for backup in all_backups:
-    print(backup)
+    print(f"{backup}")
+    print(f"{pp.pformat(backup.data)} \n")
 
-print("\n\n get_all with pagination")
+
+print("\nget_all with pagination")
 pagination = backups.get_all(limit=105, pagination=True, page_size=50)
 end = False
+
+
 while not end:
     data = pagination.data
-    print("Page size:", len(data["resources"]))
-    print(data)
+    count = len(data["resources"])
+    print(f"Page size: {count}")
+    print(f"{pp.pformat(data)}")
 
     try:
         pagination.next_page()
     except HPESimpliVityException:
         end = True
 
-print("\n\n get_by_id")
+print("\nget_by_id")
 backup = backups.get_by_id(backup_object.data["id"])
-print(backup, backup.data)
+print(f"{backup}")
+print(f"{pp.pformat(backup.data)} \n")
+
 
-print("\n\n get_by_name")
+print("\nget_by_name")
 backup = backups.get_by_name(backup_object.data["name"])
-print(backup, backup.data)
+print(f"{backup}")
+print(f"{pp.pformat(backup.data)} \n")
+
+
+vms = machines.get_all()
+if len(vms) > 0:
+    vm = vms[0]
+    backup = vm.create_backup("backup_test_from_sdk_" + str(time.time()))
+    print(f"{backup}")
+    print(f"{pp.pformat(backup.data)} \n")
+
+    # Create the new virtual machine on the same datastore
+    restored_vm = backup.restore(False, "Restored_Machine_SDK_Same_DS")
+    print(f"{restored_vm}")
+    print(f"{pp.pformat(restored_vm.data)} \n")
+
+    # Resets the original virtual machine to the same state it was in when the backup was created
+    restored_vm = backup.restore(True)
```

Comment:
We agree with this, however in order to have this "do no harm" policy we would need following changes, 
Ask user to provide virtual machine name in so that we can get the VM by that name and perform the backup and restore. 
Also this being sample example it is very unlikely that it would be executed on Production system so that way it would not be doing any harm to the system.

@sijeesh Please let us know your views on it and way to move forward.

Reply:
As per the current check-in VM and datatsore name is in variable now the changes are 'do no harm' resolving the conversation